tests/sentry/lang/javascript/test_processor.py

```diff
@@ -999,21 +998,21 @@ def test_simple(self):
 class FetchSourcemapTest(TestCase):
     def test_simple_base64(self):
         smap_view = fetch_sourcemap(base64_sourcemap)
-        tokens = [SourceMapTokenMatch(0, 0, 1, 0, src="/test.js", src_id=0)]
+        token = smap_view.lookup(1, 1, 0)
```

Comment:
where are the 3 parameters coming from? I thought we only have line + column?

Reply:
`line`, `column` and `lines_context`